src/components/Search/CategoryFilter.tsx

```diff
@@ -1,20 +1,30 @@
-import React, { useMemo } from 'react'
+import React from 'react'
 
-import { useLocation } from '@reach/router'
 import isEqual from 'lodash/isEqual'
 import toSnakeCase from 'lodash/snakeCase'
 
 import { NewGrantCategory, OldGrantCategory, SubtypeAlternativeOptions } from '../../entities/Grant/types'
 import { ProposalType, toProposalType } from '../../entities/Proposal/types'
 import { getUrlFilters } from '../../helpers'
 import useFormatMessage from '../../hooks/useFormatMessage'
+import useURLSearchParams from '../../hooks/useURLSearchParams'
 import CategoryOption from '../Category/CategoryOption'
 
 import './CategoryFilter.css'
 import CollapsibleFilter from './CollapsibleFilter'
 
-export type FilterType = typeof ProposalType | typeof NewGrantCategory | typeof OldGrantCategory
-export type Counter = Record<any, number>
+export enum ProjectTypeFilter {
```

Comment:
This could be ```All & ProjectCategory```

Reply:
🤔  this is the enum of filters. All these are then run on a `.map` and shown on the UI.